pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -2846,39 +2845,6 @@ var _ = Describe("Converter", func() {
 				Expect(domainSpec.OS.BootLoader).To(BeNil())
 				Expect(domainSpec.OS.NVRam).To(BeNil())
 			})
-
-			It("should configure the EFI bootloader if EFI insecure option", func() {
-
-				vmi.Spec.Domain.Firmware = &v1.Firmware{
-					Bootloader: &v1.Bootloader{
-						EFI: &v1.EFI{
-							SecureBoot: False(),
-						},
-					},
-				}
-				domainSpec := vmiToDomainXMLToDomainSpec(vmi, c)
-				Expect(domainSpec.OS.BootLoader.ReadOnly).To(Equal("yes"))
-				Expect(domainSpec.OS.BootLoader.Type).To(Equal("pflash"))
-				Expect(domainSpec.OS.BootLoader.Secure).To(Equal("no"))
-				Expect(path.Base(domainSpec.OS.BootLoader.Path)).To(Equal(EFICode))
-				Expect(path.Base(domainSpec.OS.NVRam.Template)).To(Equal(EFIVars))
-				Expect(domainSpec.OS.NVRam.NVRam).To(Equal("/tmp/mynamespace_testvmi"))
-			})
```

Comment:
create a tmp dir, set that as the c.OVMFPath, then create fake files to ensure the tests pass/fail as desired.

Reply:
Great tip! thank you! I've updated the PR with the new unit tests.